pallets/collator-selection/src/lib.rs

```diff
@@ -312,6 +316,8 @@ pub mod pallet {
 			Ok(().into())
 		}
 
+		/// Set the amount to deposit for becoming a collator.
+		/// When a collator calls `leave_intent` they receive the deposit back.
```

Comment:
I think this should be on the `CandidacyBond` storage item, not the function that sets it.